createdisk.sh

```diff
@@ -54,6 +54,10 @@ function create_qemu_image {
     ${QEMU_IMG} create -o lazy_refcounts=on -f qcow2 $destDir/${CRC_VM_NAME}.qcow2 31G
     ${VIRT_RESIZE} --expand /dev/sda3 $destDir/${VM_PREFIX}-base $destDir/${CRC_VM_NAME}.qcow2
 
+    # Make sure you have enough space to tmp to run VIRT_SPARSIFY
+    # Read limitation section of `man virt-sparsify`.
+    ${VIRT_SPARSIFY} --in-place $destDir/${CRC_VM_NAME}.qcow2
```

Comment:
Previous code used to set TMPDIR to the current dir:
```
    # TMPDIR must point at a directory with as much free space as the size of the
    # image we want to sparsify
    TMPDIR=$(pwd)/$destDir ${VIRT_SPARSIFY} $destDir/${VM_PREFIX}-base $destDir/${CRC_VM_NAME}.qcow2
    rm -fr $destDir/.guestfs-*
```
https://github.com/code-ready/snc/commit/f0768092dc2f0ffc78853dc2165ff109f0e5a0bf

Reply:
@cfergeau right but right now we are using the `--inplace` which doesn't work with `tmp` dir. I did try using `--tmp` option along with `--in-place` and that error out.

Reply:
setting `TMPDIR` should not error out. However, 
`If you are using the --in-place option, then large amounts of temporary space are not required.` so I'm not sure the comment you added is needed at all?

Reply:
@cfergeau I just put `enough space` not large amount which you are right required only if we don't use the `inplace` option.

Reply:
"to tmp" -> "in /tmp" if you want to keep the comment

Reply:
I can not judge about this, as this is not something I can test now